main/guides/js-programming/ses/ses-reference.md

```diff
@@ -0,0 +1,316 @@
+# SES Reference
```

Comment:
This doesn't seem to appear in the TOC. I can navigate to it via links, but then I see no relevant context in the sidebar.